modules/ROOT/partials/configuration/fullscreen_native.adoc

```diff
@@ -32,5 +32,5 @@ tinymce.init({
 The `+fullscreen_native+` option has the following limitations. When `+fullscreen_native+` is enabled:
 
 * The Escape keyboard key (Esc) will exit full screen mode. Default editor behavior of the Esc key will be overridden by browser and the Esc key will exit full screen mode instead of closing dialogs, menus, or moving focus from the editor UI to the editor content. This may lead to accessibility issues.
-* Firefox users - The full screen keyboard shortcut (Ctrl+Shift+F) cannot be used to repeatedly toggle the browser-native full screen mode on and off without interacting with the editor between on toggles. If the user does not interact with the editor between off and on toggles, the plugin will fall-back to filling the browser window.
+* For Firefox users, the full screen keyboard shortcut (Command+Shift+F or Ctrl+Shift+F) cannot be used to repeatedly toggle the browser-native full screen mode on and off without interacting with the editor between on toggles. If the user does not interact with the editor between off and on toggles, the plugin will fall-back to filling the browser window.
```

Comment:
Is this a typo: "between on toggles" doesn't sound right?

Reply:
Oh I see ... you're using "on toggles" to say the toggles that turn it on, not as a preposition. Maybe there's a clearer term?

Reply:
`fall-back` seems like something we probably have used inconsistent hyphenation for ... :)